interface/wx/display.h

```diff
@@ -143,6 +143,46 @@ class wxDisplay
      */
     wxSize GetPPI() const;
 
+    /**
+        Returns scaling factor used by this display.
+
+        The scaling factor is the ratio between GetPPI() and GetStdPPI()
+        (it is implicitly assumed that this ratio is the same for both
+        horizontal and vertical components).
+
+        @see wxWindow::GetContentScaleFactor(), wxWindow::GetDPIScaleFactor()
+
+        @since 3.1.5
+     */
+    double GetScaleFactor() const;
+
+    /**
+        Returns default display resolution for the current platform in pixels
+        per inch.
+
+        This function mostly used internally, use GetPPI() to get the actual
+        display resolution.
+
+        Currently the standard PPI is the same in both horizontal and vertical
+        directions on all platforms and its value is 96 everywhere except under
+        Mac, where it is 72.
```

Comment:
Maybe it is better to use macOS or wxOSX here?

Reply:
yes, since it is both macOS and iOS, perhaps wxOSX would be a good alternative

Reply:
Does wxOSX include wxiOS? Maybe we should just say "Apple devices" :-/

I really don't know what's the best way to refer to macOS and iOS collectively, please let me know what do you prefer.

Reply:
It could just be "macOS and iOS". It is clear and they are not that many.

Reply:
I think Apple devices is a good idea, watchOS, tvOS would be included as well